common/metrics/runtime.go

```diff
@@ -52,6 +52,8 @@ var (
 
 	// goVersion is the current runtime version.
 	goVersion = runtime.Version()
+	// cadenceVersion is the current version of cadence
+	cadenceVersion = "0.25.x"
```

Comment:
Can we add a makefile step or some other test step to ensure this doesn't get out of sync with the tagged release? @Groxx  does this seem insane?

Reply:
I think Stephen/Ender was saying that the tag version being the last and all the way to do it is to make sure we update the release steps to update this version with every branch since the version itself isn't accessible.

Reply:
I mean something like `make test` also doing a grep for this value and ensuring it's not out of sync with the tag

Reply:
late to the game, but:
unfortunately I don't think there's a way to automate this, short of automating releases entirely... which brings other major complications too.  though we might do that eventually.

for environments that use go modules (i.e. not our internal monorepo, at least currently), we could pull this information from the module's build-time metadata, which is probably the best option.  that's always up to date.

for non-module environments... dunno.